adapters/zeroclickfraud/zeroclickfraud.go

```diff
@@ -0,0 +1,188 @@
+package zeroclickfraud
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/golang/glog"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+	"strconv"
+	"text/template"
+)
+
+type ZeroclickfraudAdapter struct {
+	EndpointTemplate template.Template
+}
+
+func (a *ZeroclickfraudAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+
+	errs := make([]error, 0, len(request.Imp))
+	headers := http.Header{
+		"Content-Type": {"application/json"},
+		"Accept":       {"application/json"},
+	}
+
+	// Pull the host and source ID info from the bidder params.
+	reqImps, err := splitImpressions(request.Imp)
+
+	if err != nil {
```

Comment:
I don't believe you need the nil check here. The built-in go helpers are nil friendly.

Reply:
I do want this check in here so I don't add to the array in case it is nil. Removing this check will append a nil in the errors array even if there aren't any errors.